### PR TITLE
Android crash fixes

### PR DESCRIPTION
**Pull request by sandstranger**
I will add again this fixes

### CODE REVIEW

File: apps/openmw/mwgui/mapwindow.cpp [hunk 1]
```diff
@@ -165,7 +165,11 @@ namespace MWGui
         , mCompass(NULL)
         , mChanged(true)
         , mFogOfWarToggled(true)
+#if defined (ANDROID) && !defined (OPENGL_ES)
+        , mFogOfWarEnabled(false)
```
Comment (kpp) [inline apps/openmw/mwgui/mapwindow.cpp]: And you get warning `unused argument`. Can you move this code where it is called? Outside ctor?

File: apps/openmw/mwphysics/physicssystem.cpp [hunk 2]
```diff
@@ -1290,8 +1290,11 @@ namespace MWPhysics
         mMovementResults.clear();
 
         mTimeAccum += dt;
+#if defined (ANDROID)
+        const float physicsDt = 1.f/30.0f;
```
Comment (kpp) [inline apps/openmw/mwphysics/physicssystem.cpp]: Can you describe these magic numbers?

File: components/myguiplatform/myguirendermanager.cpp [hunk 3]
```diff
@@ -128,8 +128,11 @@ class Drawable : public osg::Drawable {
             osg::Texture2D* texture = batch.mTexture;
             if(texture)
                 state->applyTextureAttribute(0, texture);
-
+#if defined (ANDROID) || defined (OPENGL_ES)
+            osg::GLBufferObject* bufferobject = 0;
```
Comment (kpp) [inline components/myguiplatform/myguirendermanager.cpp]: Can you comment these lines? Why can't we create `vbo->getOrCreateGLBufferObject` even if `state->isVertexBufferObjectSupported`?

File: apps/openmw/mwphysics/physicssystem.cpp (hunk 2 above)
Comment (sandstranger) [inline apps/openmw/mwphysics/physicssystem.cpp]: arm processors is shit , so it need to increase physics step

File: components/myguiplatform/myguirendermanager.cpp (hunk 3 above)
Comment (sandstranger) [inline components/myguiplatform/myguirendermanager.cpp]: it is bug in osg in gles render , for example mali t760 supports vbo , but  game crashes inside osg sources

Comment (kpp) [inline components/myguiplatform/myguirendermanager.cpp]: Will you comment it inside the code?

**Comment by sandstranger** [comment]
I close it , because it not actual anymore

**Pull request closed without merging**